libpod/adapter/runtime_remote.go

```diff
@@ -320,6 +322,129 @@ func (r *LocalRuntime) Config(name string) *libpod.ContainerConfig {
 	return &data
 
 }
+
+// PruneImages is the wrapper call for a remote-client to prune images
 func (r *LocalRuntime) PruneImages(all bool) ([]string, error) {
 	return iopodman.ImagesPrune().Call(r.Conn, all)
 }
+
+// Export is a wrapper to container export to a tarfile
+func (r *LocalRuntime) Export(name string, path_ string) error {
+	tempPath, err := iopodman.ExportContainer().Call(r.Conn, name, "")
+	if err != nil {
+		return err
+	}
+
+	outputFile, err := os.Create(path_)
+	if err != nil {
+		return err
+	}
+	defer outputFile.Close()
+
+	writer := bufio.NewWriter(outputFile)
+	defer writer.Flush()
+
+	//TODO make this something for reals
+	reply, err := iopodman.ReceiveFile().Send(r.Conn, varlink.Upgrade, tempPath, true)
+	if err != nil {
+		return err
+	}
+
+	length, _, err := reply()
+	if err != nil {
+		return err
+	}
+
+	buf4096 := make([]byte, 4096)
+	var buf []byte
+
+	toBeCopied := int(length)
+
+	for toBeCopied > 0 {
+		if toBeCopied >= 4096 {
+			buf = buf4096
+		} else {
+			buf = make([]byte, toBeCopied)
+		}
+
+		numBytesRead, err := r.Conn.Reader.Read(buf)
+
+		if numBytesRead == toBeCopied {
+			err = nil
```

Comment:
Is this actually correct? What errors are we throwing out here?

Reply:
used copyn thanks!